tensorflow/stream_executor/cuda/cuda_diagnostics.cc

```diff
@@ -315,8 +315,7 @@ port::StatusOr<DriverVersion> Diagnostician::FindKernelDriverVersion() {
     // NOTE: OSX CUDA driver does not currently store the same driver version
     // in kCFBundleVersionKey as is returned by cuDriverGetVersion
     CFRelease(kext_infos);
-    const CFStringRef str = (CFStringRef)CFDictionaryGetValue(cuda_driver_info,
-							 kCFBundleVersionKey);
+    const CFStringRef str = (CFStringRef)CFDictionaryGetValue(cuda_driver_info, kCFBundleVersionKey);
```

Comment:
So the default alignment looks like below, which is also more than 80 chars
```
    const CFStringRef str = (CFStringRef)CFDictionaryGetValue(cuda_driver_info,
							                                  kCFBundleVersionKey);

```

This Mac API is like Java, kind of a pain to get to fit

An alternative could be

```
    const CFStringRef str =
      (CFStringRef)CFDictionaryGetValue(cuda_driver_info, kCFBundleVersionKey);

```